card/lib/card/format/nest/fetch.rb

```diff
@@ -1,55 +1,58 @@
 class Card
   class Format
-    module Nest
+    class Nest
```

Comment:
biggest change is introduction of this new class, which reduced the confusion from passing variables all around.

I think this class should get even simpler when modal handling is moved to Card::View, where it belongs.